gapis/resolve/resource_data.go

```diff
@@ -180,6 +180,19 @@ func (r *ResourceDataResolvable) Resolve(ctx context.Context) (interface{}, erro
 // Pipelines resolves the data of the currently bound pipelines at the specified
 // point in the capture.
 func Pipelines(ctx context.Context, p *path.Pipelines, r *path.ResolveConfig) (interface{}, error) {
+	cmd, err := Cmd(ctx, p.After, r)
+	if err != nil {
+		return nil, err
+	}
+	state, err := GlobalState(ctx, &path.GlobalState{After: p.After}, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if !cmd.CmdFlags(ctx, api.CmdID(p.After.Indices[0]), state).IsExecutedDraw() {
+		return nil, fmt.Errorf("Select a draw call.")
```

Comment:
This is roughly the same message as Messages.SELECT_DRAW_CALL, but it includes the red error icon. I am open to a solution that doesn't duplicate this string.

Reply:
Anything in the UI should be using messages from the messages file.
gapis/messages/en-us.stb.md

Reply:
I think this is still not QUITE what you want. Since I think it will grab ALL piplines
when you click any draw-call. This will run into the same problems as just clicking on a random command.

I wonder if we could just ONLY present bound pipelines? (That is sad because it prevents us from inspecting other pipelines in the state).

Then we would just return NO pipelines for the  case where you have not clicked on something valid.

Also is does IsExecutedDraw return true for a top-level VkCmdDraw() call? Which is not itself a draw-call but a recording of one?

Reply:
I think this will also drop all Dispatch calls?

Reply:
I am not sure what you are asking me with dispatches. They are also decorated with @executed_draw in the API files.

Reply:
"I wonder if we could just ONLY present bound pipelines? (That is sad because it prevents us from inspecting other pipelines in the state).

Then we would just return NO pipelines for the case where you have not clicked on something valid."

This would still show pipelines between draws, which I believe we don't want